cdc/sink/producer/kafka/kafka.go

```diff
@@ -116,8 +133,7 @@ func (k *kafkaSaramaProducer) SendMessage(ctx context.Context, message *codec.MQ
 		return ctx.Err()
 	case <-k.closeCh:
 		return nil
-	default:
-		k.asyncClient.Input() <- msg
+	case k.asyncClient.Input() <- msg:
```

Comment:
`SyncBroadcastMessage` has a similar default branch, please check whether we need to update in `SyncBroadcastMessage`